Cargo.lock

```diff
@@ -1120,38 +1114,14 @@ dependencies = [
  "zeroize",
 ]
 
-[[package]]
-name = "darling"
```

Comment:
Harvesting the fruit of upstream work here: https://github.com/najamelan/futures_ringbuf/pull/9